var/spack/repos/builtin/packages/py-pycairo/package.py

```diff
@@ -13,14 +13,16 @@ class PyPycairo(PythonPackage):
     homepage = "https://www.cairographics.org/pycairo/"
     pypi = "pycairo/pycairo-1.17.1.tar.gz"
 
+    version('1.20.0', sha256='5695a10cb7f9ae0d01f665b56602a845b0a8cb17e2123bfece10c2e58552468c')
     version('1.18.1', sha256='70172e58b6bad7572a3518c26729b074acdde15e6fee6cbab6d3528ad552b786')
     version('1.17.1', sha256='0f0a35ec923d87bc495f6753b1e540fd046d95db56a35250c44089fbce03b698')
 
-    depends_on('cairo@1.13.1:')
+    depends_on('cairo@1.13.1: +pdf')
```

Comment:
The latest version requires `cairo@1.15.10:`

Reply:
Got it, update the patch in:
https://github.com/spack/spack/pull/21004/commits/3dc3740451fb335b37bb53d2dc46e0bed482e129